Dockerfile

```diff
@@ -23,7 +23,7 @@ COPY --from=builder /build/dist/*.whl /home/datashader/tmp/
 ENV PATH="$PATH:/home/datashader/.local/bin"
 RUN pip install --upgrade pip && \
     pip install --no-cache-dir /home/datashader/tmp/*.whl && \
-    pip install gunicorn==21.2.0 && \
+    pip install gunicorn==20.1.0 && \
```

Comment:
Is going back in version going to fix the ssl error we were seeing earlier? or why are you going back in version?